css/style.css

```diff
@@ -0,0 +1,918 @@
+* {
+  box-sizing: border-box;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  background-color: #eee;
+  z-index: 2;
+}
+
+.tablet,
+.nav1,
+.nav2 {
+  display: none;
+}
```

Comment:
Same styles of the selector as line 103. In order to minimize code, you can group selectors with the same style definition.